### ISSUE TITLE
Maybe GeoFence issue.

### ISSUE THREAD

**Issue by whmoorejr** [FlyingDiver/Indigo-GeoFence#6]
I'm looking at another geospacial app to leverage kid location awareness... Owntracks.  It can send notifications via HTTP or via MQTT.  The GeoFence plugin sees the HTTP notifications and thinks it's geohopper information, but missing parameters.  The app has "Expert Mode" to further manipulate some of the data (subTopic, clientId, etc) but I have no idea what to mess with or if it will help.

On the otherhand, is this a better time for me to finally figure out MQTT and look at some of your other plugins (broker, connector, shims)... in that case, I just need to spend more time on the forum figuring out what I'm doing there.

If you think this app is a good fit for the GeoFence plugin, I've attached the debug log from 2 different actions (a periodic timed update "t":"t" and a manual user update "t":"u" ).  In HTTP, enter/exit is one code "t":"c" for enter/exit a circle or "t":"b" for enter/exit a beacon.

Basically, looking to you for advice on which is the way to go... manipulate the app or geofence plugin or dive into MQTT.  
[OwnTracks Debug.txt](https://github.com/FlyingDiver/Indigo-GeoFence/files/13907810/OwnTracks.Debug.txt)

**Comment by FlyingDiver** [comment]
Do you have a link to their API docs?

**Comment by FlyingDiver** [comment]
If you wanted to try it via MQTT you would just need the Broker and Connector.  I would just trigger off the raw messages, and not try to do a Shim Device.  Unless you really need a device for each person, rather than variables or what not.

**Comment by whmoorejr** [comment]
https://owntracks.org/booklet/tech/json/

Respectfully,

Bill Moore
***@***.***
832-294-3096

**Comment by whmoorejr** [comment]
owntracks looks like a pretty good tool.  I'm working on setting it up as individual devices (One per phone).  I'm getting pretty close to it being awesome.  I have the app pushing a timestamp, battery level, on wifi or cell, lattitude, longitude, and an array of any regions (geofences) the phone is in and the trigger cause (enter/exit/user/time/etc.).  Not as clean for just knowing if I'm home or not, but good for knowing where a device is when it's not in a fence. (kids don't like to stay within fences).  A few hickups I'm working through.... did I set up the shim correctly?  The shim device is populated with all of the payload, but I need to tweek it some... like lattitude and longitude get rounded 2 past decimal.  The other is security.  I have it running fine internally in my home (no password added), but I need to access from outside... password or API key.
